docs/examples/cookbook/drag-and-drop.md

```diff
@@ -436,7 +436,7 @@ position of the `Draggable`, that offset can be applied to keep the UI fixed
 in position relative to the mouse.
 
 ```Lua
-		Position = scope:Computed(function(use)
+		Position = scope:Computed(function(use, scope)
```

Comment:
Scope shouldn't be provided as it is never used
```suggestion
		Position = scope:Computed(function(use)
```

Reply:
reverted